tests/gnrc_udp/Makefile

```diff
@@ -8,7 +8,6 @@ USEMODULE += gnrc_pktbuf_cmd
 USEMODULE += gnrc_udp
 USEMODULE += gnrc_rpl
 USEMODULE += auto_init_gnrc_rpl
-USEMODULE += od
```

Comment:
Do you mind removing it from all the other tests selecting `gnrc_pktbuf_cmd` that do not need to select `od` since they do not use it directly?

Reply:
Some of them include it for other reasons and this needs to be carefully checked. Can we do it as a follow-up?

Reply:
Sure